PontoFacil/PontoFacil/Models/ClockIn.cs

```diff
@@ -38,7 +38,7 @@ public void Close(DateTime dt)
 
         public bool IsOpen()
         {
-            return _start != null;
+            return _start != null && (_end == null || DateTime.MinValue == _end );
```

Comment:
_start should be checked like this _start != DateTime.MinValue, cause DateTime could not be null.